packages/svelte-ux/src/lib/components/setup.ts

```diff
@@ -0,0 +1,35 @@
+import type { FormatNumberOptions } from '$lib/utils/number';
+import { getContext, setContext } from 'svelte';
+import type { Theme } from './theme';
+
+export type Setup = {
```

Comment:
I think I'd like to call this `Settings` instead of `Setup` (everywhere, not just this instance).

I could see apply an override of settings somewhere in your component tree (since it's just context).  For example maybe if you had routing with the locale (`/en/...`, `/fr/...`, etc), or you wanted to style a page/section different (frontpage marketing vs app), etc.

It would also be nice to have a `<Settings>` component like `<Theme>` (which just makes it easier to set the context)

Reply:
All right, I'll update to `Settings`.

The override thing, you can already do it no?